.gitignore

```diff
@@ -5,3 +5,10 @@
 /package/pkg
 /package/src
 /media/*.pdf
+
+# Tests
```

Comment:
Let me know if the following three lines are necessary, because I suspect they were cherry-picked accidentally.

Reply:
If so, I'll remove them in the next PR.